Python/bltinmodule.c

```diff
@@ -2516,6 +2516,41 @@ Without arguments, equivalent to locals().\n\
 With an argument, equivalent to object.__dict__.");
 
 
+/* Improved Kahan–Babuška algorithm by Arnold Neumaier
+   Neumaier, A. (1974), Rundungsfehleranalyse einiger Verfahren
+   zur Summation endlicher Summen.  Z. angew. Math. Mech.,
+   54: 39-51. https://doi.org/10.1002/zamm.19740540106
+   https://en.wikipedia.org/wiki/Kahan_summation_algorithm#Further_enhancements
+ */
+
+typedef struct {
+    double sum;  /* accumulator */
+    double c;  /* a running compensation for lost low-order bits */
```

Comment:
```suggestion
    double sum;  /* accumulator */
    double c;    /* a running compensation for lost low-order bits */
```

(Not sure if you do that kind of style)